modules/hitl/src/migrations/v12_13_0-1605201510-add_thread_id_column_to_session_table.ts

```diff
@@ -0,0 +1,28 @@
+import * as sdk from 'botpress/sdk'
+
+const migration: sdk.ModuleMigration = {
```

Comment:
I also added this migration, which targets 12.13.0.
This is the exact same migration as `modules/hitl/src/migrations/v12_10_8-1600900478-add_thread_id_column_to_session_table.ts`.

This will be useful for clients that upgrade to 12.13.0 and are missing the `thread_id` column, but don't want to add the column manually by connecting to their database.

Reply:
If the logic is exactly the same, then can you just import and export with a comment, so it's clearer? Ex:
```js
import prevMigration from 'v12_10_08'
// re-executing migration because.. 
export default prevMigration
```

Reply:
Good idea, will try that!